lms/djangoapps/courseware/model_data.py

```diff
@@ -34,12 +35,473 @@ class InvalidWriteError(Exception):
     """
 
 
-def chunks(items, chunk_size):
+def _all_usage_keys(descriptors, aside_types):
     """
-    Yields the values from items in chunks of size chunk_size
+    Return a set of all usage_ids for the descriptors that this FieldDataCache is caching
```

Comment:
This method has no direct connection to `FieldDataCache` - probably shouldn't even be mentioned in the docstring.